kivymd/theming.py

```diff
@@ -636,29 +658,33 @@ def set_colors(self, *args) -> None:
             else:
                 self._set_palette_color()
         else:
-            path_to_wallpaper = get_wallpaper(
-                App.get_running_app().user_data_dir, self.path_to_wallpaper
+            system_scheme = get_dynamic_scheme(
+                dark_mode=False if self.theme_style == "Light" else True,
```

Comment:
A utility function/method for this would be nice 👍🏼

Reply:
Yes this would forward all platform related issues to [materialyoucolor-python](https://github.com/T-Dynamos/materialyoucolor-python) instead of KivyMD.

Reply:
Ah, I meant a helper function for this statement `False if self.theme_style == "Light" else True` as this is used twice in the codebase...
Just my suggestion, if it's overkill, just ignore this 🙂